tests/testthat/test-occ_issues.R

```diff
@@ -4,16 +4,16 @@ test_that("occ_issues", {
   skip_on_cran() # because fixture in .Rbuildignore
 
   vcr::use_cassette("occ_issues", {
-    out <- occ_search(limit = 100)
 
     # Parsing output by issue
     res <- occ_search(
-      geometry = 'POLYGON((30.1 10.1, 10 20, 20 40, 40 40, 30.1 10.1))',
```

Comment:
Had to change this because test was failing because it did not return any issues. Thought it would be better to fetch data that we know has the issues we test for.